### DISCUSSION TITLE
Suggest a better name.

### DISCUSSION THREAD

**Discussion by Kaustubh-Natuskar** [Kaustubh-Natuskar/companies-to-apply#134] [Ideas]
Hello everyone, when I started this project i assigned a random name to this repository. 
But now i think we should change it to something non-generic so that it will be easier to remember. 
Please suggest some good name 🙂.
We also have opened a issue  _#130_  for this. You can comment down below or in that issue's comment section.
Thank you.

**Comment by being-nitin** [comment]
Exceptional proji.

**Comment by hasnatkhan786** [comment]
Should be coding guru [😁](https://balloonslane.com/)

**Comment by mdsahil368828** [comment]
[https://eliteluxuryyacht.com/](https://eliteluxuryyacht.com/)
